internal/seed/config/populate.go

```diff
@@ -18,14 +18,14 @@ import (
 	"bufio"
 	"errors"
 	"fmt"
+	"github.com/edgexfoundry/go-mod-registry/registry"
```

Comment:
Can you move this import to be with the others?

Reply:
Yep. @tsconn23 do you know any setting in Goland for it not insert imports in the std lib section. Thx

Reply:
I haven't played around with it, but there's a setting under `Tools --> Go Tools --> Goimports file` that could help. Also see https://godoc.org/golang.org/x/tools/cmd/goimports